pkg/app/app_reconcile.go

```diff
@@ -60,7 +60,19 @@ func (a *App) reconcileDelete() error {
 
 	a.resetLastDeployStartedAt()
 
-	result := a.delete(a.updateLastDeployNoReturn)
+	var result exec.CmdRunResult
+	if !a.app.Spec.NoopDelete {
```

Comment:
typically if we are working with both if branches we should deal with positive case first to avoid unnecessary negation (extra code reading burden)